dot_parser.py

```diff
@@ -548,8 +548,13 @@ def parse_dot_data(s):
         tokens = graphparser.parseString(s)
         return list(tokens)
     except ParseException as err:
-        print(
-            err.line +
-            " "*(err.column-1) + "^" +
-            err)
+        if PY3 and (hasattr(ParseException, 'explain') and
+                    callable(getattr(ParseException, 'explain'))):
+            print(ParseException.explain(err))
+        else:
+            print(
```

Comment:
See comment about `stderr`, `logging`, above.

Reply:
Here as well, you are right. Following our discussion in https://github.com/pydot/pydot/issues/171#issuecomment-622489506, I will use logging instead.